views/components/ConfigurationsList/index.jsx

```diff
@@ -382,31 +245,17 @@ const ConfigurationsList = ({ user, classes, theme }) => {
       }
     })
   }
-  const removeConfig = (configs, index, configID) => {
-    removeConfiguration(configID)
-    setState((prevState) => {
-      return {
-        ...prevState,
-        configurations: update(configs, {
-          $splice: [[index, 1]],
-        }),
-        snackTime: true,
-      }
-    })
-    if (index == state.preferences['config']) {
-      updateUserPreferences(0, 'index')
-    }
-  }
-  const openSearchUsers = (index, configID, shared, owner) => {
-    setState((prevState) => {
+
+  const openSearchUsers = (config) => {
+    const { _id, readers, owner } = config
+    setSharedWith((prevState) => {
       return {
         ...prevState,
         searchUsers: true,
         selectedConfig: {
-          _id: configID,
-          index: index,
+          _id,
         },
-        shared: shared.map((friend) => ({
+        shared: readers.map((friend) => ({
           label: friend,
           value: friend,
         })),
```

Comment:
We can get this from the `config.readers` property so we don't need it stored in state.

Reply:
This is used by the shared component it could be removed but out this is out of scope